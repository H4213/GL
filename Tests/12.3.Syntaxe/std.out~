var a;
a := 5;
ecrire a;
