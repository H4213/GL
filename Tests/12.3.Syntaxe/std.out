------ Impression du programme -------

var a;
a := 5;
ecrire a;

------- Fin du programme -------

